Source/WebCore/rendering/style/StyleRareNonInheritedData.h

```diff
@@ -213,7 +213,8 @@ class StyleRareNonInheritedData : public RefCounted<StyleRareNonInheritedData> {
 
     unsigned textDecorationStyle : 3; // TextDecorationStyle
 
-    unsigned aspectRatioType : 3;
+    unsigned aspectRatioType : 2;
+    unsigned contentVisibility : 2;
```

Comment:
Same thought on the comment, why not do it here as we do with the others?

Reply:
Ditto.